src/pineko/evolve.py

```diff
@@ -142,6 +143,13 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
     operators_card["eko_version"] = metadata.version("eko")
     pineko_version = metadata.version("pineko")
 
+    # switch on polarization, only in DIS
+    if "yadism_version" in pineappl_grid.key_values():
+        grid_runcard = pineappl_grid.key_values()["runcard"]
```

Comment:
why don't we do immediately the correct thing? i.e. attribute it to the grid
1. Adjust [`yadbox.export.dump_pineappl_to_file`](https://github.com/NNPDF/yadism/blob/d6366b5e7e078c02193c1cfa5df0639b3f9c19d5/src/yadbox/export.py#L104) by adding
```py
grid.set_key_value("polarized", obsname.startswith("g"))
```
(we could even add a method/property to `ObservableName` since I seem to remember to have seen that test before)
2. do here
```py
kv = pineappl_grid.key_values()
operators_card["configs"]["polarized"] = "polarized" in kv and bool(kv["polarized"])
```

Reply:
This is also a solution.

Reply:
and moreover without any additional effort for double hadronic cases on the pineko side

Reply:
sure but in principle yadism doesn't need that information and a single grid can contain polarised and unpolarised observables.
And same can hold for the future hadronic polarised tool.

Reply:
> sure but in principle yadism doesn't need that information and a single grid can contain polarised and unpolarised observables. 

yadism (native) output yes, [yadbox.export.dump_pineappl_to_file](https://github.com/NNPDF/yadism/blob/d6366b5e7e078c02193c1cfa5df0639b3f9c19d5/src/yadbox/export.py#L104) no

Reply:
1. [yadbox.export.dump_pineappl_to_file](https://github.com/NNPDF/yadism/blob/d6366b5e7e078c02193c1cfa5df0639b3f9c19d5/src/yadbox/export.py#L104) adjusted in: https://github.com/NNPDF/yadism/commit/508c4feb2d4a91d2c91820a25790043cf62f6faa
2. Added in https://github.com/NNPDF/pineko/pull/128/commits/ca5eca6b91ce1b3efa09ae49bbea8b3afd43aa56

Reply:
I was not fully aware of this but I like it very much this way 🙃